core/src/main/resources/jenkins/diagnosis/HsErrPidList/index_de.properties

```diff
@@ -0,0 +1,31 @@
+# The MIT License
+#
+# Copyright (c) 2017 Daniel Beck and a number of other of contributors
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+Delete=L\u00F6schen
+Date=Datum
+Name=Name
+Java\ VM\ Crash\ Reports=Absturzprotokolle der Java-VM
+blurb=Die folgenden Absturzprotokolle der Java-VM wurden f\u00FCr diese Jenkins-Instanz gefunden. \
+  Wenn Sie dies f\u00FCr ein Problem in Jenkins halten, <a href="https://jenkins.io/redirect/report-an-issue">erstellen Sie bitte einen Bugreport</a>. \
```

Comment:
Throughout `Jenkins/oops_de.properties`, the terms "Bug-Report" and "Bug-Tracker" are used, so the usage of "Bugreport" should be made consistent here.